frontend/src/pages/profile.tsx

```diff
@@ -1,7 +1,56 @@
+import { FieldValues, SubmitHandler, useForm } from "react-hook-form";
+
+import { Checkbox } from "../components/Checkbox";
+import Radio from "../components/Radio";
+import { Textfield } from "../components/Textfield";
+
 export default function Profile() {
+  const dietaryList = ["Nuts", "Eggs", "Seafood", "Pollen", "Dairy", "Other"];
+
+  const { register, setValue, handleSubmit } = useForm();
+
+  const onSubmit: SubmitHandler<FieldValues> = (data) => {
+    console.log(data);
+  };
   return (
     <main>
-      <h1>PIA Profile Page!</h1>
+      <form
+        onSubmit={handleSubmit(onSubmit)}
+        className="flex flex-col items-center justify-between gap-10 p-12"
+      >
+        <div className="grid gap-5">
+          <Textfield register={register} name={"firstname"} label="First" placeholder="John" />
+          <Textfield
+            register={register}
+            name={"email"}
+            label={"Email"}
+            type="email"
+            placeholder="johnsmith@gmail.com"
+          />
+          <Textfield
+            register={register}
+            setValue={setValue}
```

Comment:
The deployment fails because the frontend build fails (can check that by running `npm run build`).
The issue seems to be `setValue` does not exist on `Textfield`, so if you change this line to `setCalendarValue={setValue}` it'll fix the build and make the calendar popover work.